dev-tools/omdb/src/bin/omdb/db/saga.rs

```diff
@@ -142,6 +183,14 @@ async fn cmd_sagas_inject_error(
                 // If there's no current SEC, then we don't need to check if
                 // it's up. Would we see this if the saga was Requested but not
                 // started?
+                let text = "warning: saga has no assigned SEC, so cannot \
+                verify that the saga is not still running!";
```

Comment:
I like these messages.  But it's too late at this point, right?  Shouldn't we print the confirmation prompt after we report any extra risks?

Reply:
oh, definitely. added a prompt in 8c3c39ba0

Reply:
Sorry, I meant that to apply to all of the warnings here.  There are I think three other places where we report that we couldn't verify the safety but we proceed anyway.

I wonder if the simplest thing is to just call `read_and_validate()` once, before making any changes, right before we make the changes (so like L321 in the current PR).  That can cover the general warning up front plus any failures to verify safety.

Reply:
> There are I think three other places where we report that we couldn't verify the safety but we proceed anyway.

Most spots do a `bail!`. Right now there's a y/n prompt:

- at the beginning after the large warning
- if the SEC is None
- after displaying the ping's `CommunicationError`

We can elide the first one, and the second and third ones amount to the two ways to exit the `match` on L181, so yeah - d45919075 has one confirmation prompt :)

Reply:
Ah, sorry I misread those paths.